src/internal/const.js

```diff
@@ -8,3 +8,9 @@ export const FL = {
 };
 
 export const ordinal = ['first', 'second', 'third', 'fourth', 'fifth'];
+
+export const namespace = 'fluture';
+export const name = 'Future';
+export const version = 2;
```

Comment:
Version `2` because all Futures between 1.0.0 and 5.0.0 are conceptually version `1` (even though never explicitly specified). The version number changes when a change to the *class surface* is made. For example, all futures before 6.0.0 had an `_f`-method which Fluture expects and makes use of. Since 6.0.0 this has been renamed to `_fork`. Fluture 6.0.0 is therefore (and for more subtle reasons to do with the way Actions are implemented) incapable of consuming older Fluture Futures.